robot/markers.py

```diff
@@ -168,9 +168,12 @@ def spherical(self) -> SphericalCoord:
         """
         return SphericalCoord(*self._raw_data['spherical'])
 
+    def __repr__(self):
+        return "<{}>".format(str(self))
```

Comment:
Please drop redundant `str` call here (as above).

Reply:
half-fixed (& broken something unrelated) in 7f9cfee 
Then undid the stupid breakage in 515e0a3 (not sure what I was on when I thought I fixed that.)
Then the other half is fixed in d4e364e